.platform.app.yaml

```diff
@@ -177,7 +177,7 @@ workers:
         disk: 512
         commands:
             start: |
-                php /app/bin/console akeneo:batch:job-queue-consumer-daemon
+                php /app/bin/console messenger:consume ui_job import_export_job data_maintenance_job --env=prod
```

Comment:
I believe we should also include the events api by configuring the `webhook` queue here as well https://docs.akeneo.com/master/install_pim/manual/events_api.html